File: flow004.java
/*Write a program which will show the sum of first and last digit
of the number*/
import java.io.IOException;
import java.io.BufferedReader;
import java.io.InputStreamReader;
class flow004 
{
	public static void main(String[] args) throws IOException 
	{
		BufferedReader inp=new BufferedReader(new InputStreamReader(System.in));
		int testCases,num,firstDigit,lastDigit,sum;
		testCases=Integer.parseInt(inp.readLine());
		while(testCases>0)
		{
			sum=0;
			num=Integer.parseInt(inp.readLine());
			if(num%10==0 || num<10)
			{
               if(num<10)
               	sum=num;
               else
               	while(num>=10)
               		num/=10;
               	sum=num;
               
			}
			else
			{
				sum+=num%10;
				while(num>10)
					num/=10;
				sum+=num;
			}
			System.out.println(sum);
			testCases--;
		}
	}
}

File: flow004.cpp
#include<iostream>
using namespace std;
#include<string>
int value(int);
int main()
{
	int testCases;
	cin>>testCases;
    while(testCases--)
    {
    	int number;
    	cin>>number;
    	cout<<value(number)<<endl;
    }
    return 0;
}

int value(int a)
{
	int sum=0;
	string mess;
	mess=to_string(a);
	sum=(mess[0]-'0')+(mess[mess.length()-1]-'0');
	return sum;
}